### ISSUE TITLE
Pagination not working, using Elementor version 3.5.5 and Elementor pro version 3.6.0

### ISSUE THREAD

**Issue by hrvojelee** [dudaster/ele-custom-skin#690]
Hey team,

Paginations is not working with the numbers +previous/next
![pagination](https://user-images.githubusercontent.com/99299218/153097169-7bc6f649-c531-462d-af87-fbd6d083e04a.png)
. I have built a custom loop and only how I can preview the next page is with the load more button which is not something that we want.

Can you please check that issue? I'm building a site with elementor builder.

**Comment by PPjev** [comment]
Same problem persists for many years I guess :(

**Comment by hrvojelee** [comment]
well for a paid plugin that's not good. In this case I'm using a free one but I want to buy it, but if this problem exists for years that's bad. It works on older install of Elementor, just this new update it stopped working.

**Comment by dudaster** [comment]
We are working on a fix.

**Comment by hrvojelee** [comment]
I appreciate it guys! You maybe have some eta on the update of the plug? Would love to tell my clients what to expect? As for right now I have set load more button.

**Comment by oshibobo** [comment]
Pagination not working

**Comment by tracystone** [comment]
This is still an issue.

**Comment by albusayn** [comment]
yes still not working its taking over the native elementor load more and continious loading. In this way we cant use it. Ele custom is taking over this.
